product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfigCompositeApplication/pom.xml

```diff
@@ -59,141 +61,135 @@
+      <artifactId>SOAP_EP</artifactId>
       <version>1.0.0</version>
-      <type>zip</type>
+      <type>xml</type>
     </dependency>
     <dependency>
-      <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_json_to_soap</artifactId>
+      <groupId>org.wso2.ei.product-scenarios.scenario_01.endpoint</groupId>
+      <artifactId>JSON_EP</artifactId>
       <version>1.0.0</version>
-      <type>zip</type>
+      <type>xml</type>
     </dependency>
     <dependency>
-      <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_json_to_soap_inputSchema</artifactId>
+      <groupId>org.wso2.ei.product-scenarios.scenario_01.proxy-service</groupId>
+      <artifactId>1_6_2_1_Proxy_Add_Sibling_Inline</artifactId>
       <version>1.0.0</version>
-      <type>zip</type>
+      <type>xml</type>
     </dependency>
     <dependency>
       <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_json_to_soap_outputSchema</artifactId>
+      <artifactId>config_1.6.3.7_xml</artifactId>
       <version>1.0.0</version>
       <type>zip</type>
     </dependency>
     <dependency>
       <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_json_to_soap_xsltStyleSheet</artifactId>
+      <artifactId>config_1.6.3.6_xml</artifactId>
       <version>1.0.0</version>
       <type>zip</type>
     </dependency>
     <dependency>
       <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_soap_to_json</artifactId>
+      <artifactId>config_convert_json_to_soap_xsltStyleSheet</artifactId>
       <version>1.0.0</version>
       <type>zip</type>
     </dependency>
     <dependency>
-      <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_soap_to_json_inputSchema</artifactId>
+      <groupId>org.wso2.ei.product-scenarios.scenario_01.endpoint</groupId>
+      <artifactId>POX_EP</artifactId>
       <version>1.0.0</version>
-      <type>zip</type>
+      <type>xml</type>
     </dependency>
     <dependency>
-      <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_soap_to_json_outputSchema</artifactId>
+      <groupId>org.wso2.ei.product-scenarios.scenario_01.proxy-service</groupId>
+      <artifactId>1_6_2_2_Proxy_Add_Sibling_xpath</artifactId>
       <version>1.0.0</version>
-      <type>zip</type>
+      <type>xml</type>
     </dependency>
     <dependency>
-      <groupId>org.wso2.ei.product-scenarios.scenario_01.resource</groupId>
-      <artifactId>config_convert_soap_to_json_xsltStyleSheet</artifactId>
+      <groupId>org.wso2.ei.product-scenarios.scenario_01.proxy-service</groupId>
+      <artifactId>1_6_10_2_Proxy_AlterPayloadWithInlineJavaScript</artifactId>
```

Comment:
It seems like the diff is not only for the added tests? Shall we revisit and only have the relevent diff?